services/web/server/src/simcore_service_webserver/computation.py

```diff
@@ -26,6 +27,10 @@
     __name__, ModuleCategory.ADDON, config_section=CONFIG_SECTION_NAME, logger=log
 )
 def setup(app: web.Application):
+    # init settings
+    cfg = ComputationSettings.create_default()
```

Comment:
MINOR: i thought we would name this create function ``create_from_env()``. see [here](https://github.com/ITISFoundation/osparc-simcore/blob/f675b36e0fefe31964b2295e50f832bd4691470e/services/director-v2/src/simcore_service_director_v2/core/settings.py#L154)

Reply:
ahah man.. I copy pasted your own code... so I guess you changed it. nevermind, will change it

Reply:
done